lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButtonsRow.kt

```diff
@@ -95,45 +91,81 @@ fun OdsIconToggleButtonsRow(
     }
 }
 
-data class OdsIconToggleButtonsRowItem(
-    val icon: Painter,
-    val iconDescription: String,
-    val enabled: Boolean = true
-)
+/**
+ * An icon of an [OdsIconToggleButtonsRow].
+ */
+class OdsIconToggleButtonsRowIcon : OdsComponentIcon {
 
-@Composable
-private fun IconToggleButtonsRowItem(
-    index: Int,
-    iconToggleButton: OdsIconToggleButtonsRowItem,
-    selected: Boolean,
-    displaySurface: OdsDisplaySurface,
-    onClick: (Int) -> Unit
-) {
-    val iconTint = buttonToggleIconColor(displaySurface, selected, iconToggleButton.enabled)
-    val backgroundAlpha = if (selected && iconToggleButton.enabled) 0.12f else 0f
-    val iconToggleButtonStateDescription = selectionStateDescription(selected = selected)
-
-    Icon(
-        modifier = Modifier
-            .background(
-                color = buttonToggleBackgroundColor(displaySurface)
-                    .copy(alpha = if (iconToggleButton.enabled) animateFloatAsState(backgroundAlpha, label = "").value else backgroundAlpha)
-            )
-            .padding(12.dp)
-            .run {
-                if (iconToggleButton.enabled) {
-                    clickable(interactionSource = remember { DisabledInteractionSource() }, indication = null) { onClick(index) }
-                } else {
-                    this
+    /**
+     * Creates an instance of [OdsIconToggleButtonsRowIcon].
+     *
+     * @param painter Painter of the icon.
+     * @param contentDescription The content description associated to this [OdsIconButtonIcon].
```

Comment:
```suggestion
     * @param contentDescription The content description associated to this [OdsIconToggleButtonsRowIcon].
```

The same suggestion can be applied to other constructors.